service does not start on at least 2 laptops

## 🐛 Describe the bug
<!-- A clear and concise description of what the bug is. -->
When starting the application raise errors of incompatibilities and does not actually start.

## 📜 To Reproduce
Steps to reproduce the behavior:
1. Follow the deployment procedure using the main branch on github
2. npm start

See error in terminal:
```
Failed to compile.

Module not found: Error: Can't resolve 'crypto' in '/Users/loubrieu/tmp/doi-ui/node_modules/react-oauth2-pkce/dist'
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "crypto": require.resolve("crypto-browserify") }'
	- install 'crypto-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "crypto": false }
WARNING in ./node_modules/sax/lib/sax.js 139:13-37
Module not found: Error: Can't resolve 'stream' in '/Users/loubrieu/tmp/doi-ui/node_modules/sax/lib'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "stream": require.resolve("stream-browserify") }'
	- install 'stream-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "stream": false }

ERROR in ./node_modules/react-oauth2-pkce/dist/index.modern.js 2:0-49
Module not found: Error: Can't resolve 'crypto' in '/Users/loubrieu/tmp/doi-ui/node_modules/react-oauth2-pkce/dist'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "crypto": require.resolve("crypto-browserify") }'
	- install 'crypto-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "crypto": false }

ERROR in ./node_modules/string_decoder/node_modules/safe-buffer/index.js 3:13-30
Module not found: Error: Can't resolve 'buffer' in '/Users/loubrieu/tmp/doi-ui/node_modules/string_decoder/node_modules/safe-buffer'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "buffer": require.resolve("buffer/") }'
	- install 'buffer'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "buffer": false }

webpack compiled with 2 errors and 1 warning


```

## 🕵️ Expected behavior
<!-- A clear and concise description of what you expected to happen -->
The application should start

## 📚 Version of Software Used
<!-- Software should have a `-V` or `--version` flag to get this information. -->
dev version on branch main


## 🩺 Test Data / Additional context
<!-- If applicable, Add test data or any other context about the problem here -->

##  🏞Screenshots
<!-- If applicable, add screenshots to help explain your problem. -->

## 🖥 System Info
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

---
<!-- FOR DEV TEAM USE -->

## 🦄 Related requirements


## ⚙️ Engineering Details

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "react-oauth2-pkce": "^2.0.7",
     "react-redux": "^7.2.4",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "5.0.1",
+    "react-scripts": "4.0.3",
     "redux": "^4.1.0",
     "redux-saga": "^1.1.3",
     "styled-components": "^5.3.5",
@@ -28,7 +28,7 @@
     "xml-js": "^1.6.11"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject"

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "xml-js": "^1.6.11"
   },
   "scripts": {
-    "start": "react-scripts --openssl-legacy-provider start",
+    "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject"

